src/mrinufft/trajectories/tools.py

```diff
@@ -555,24 +613,43 @@ def get_gradient_amplitudes_to_travel_for_set_time(
             gmax=gmax,
             smax=smax,
         )
-        N = (
-            np.max(np.sum(n_ramp_down + n_ramp_up + n_plateau, axis=0)) + 2
-        )  # Extra 2 buffer samples
+        N = np.max(n_ramp_down + n_ramp_up + n_plateau) + 2
 
```

Comment:
This change is actually a bug in previous PR. We need to check for maximum time needed, not the sum across dimensions!